### DISCUSSION TITLE
gitkraken

### DISCUSSION THREAD

**Discussion by mahmim** [community/community#38100] [New to GitHub]
Hi,

I am a student at DCI institut in Germany and have a pro version Github account. but I still can't install a pro version of gitkraken. I talked about this problem with IDCI T support but they could couldn't help.I tried two times to install Gitkraken but I am only authorized to install trial version. I am the only student at DCI who has encountered this problem.

Thank you in advance for your help

Mahshid Tavallai

**Comment by ossanna16** [comment]
Hi there @mahmim and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
